### PR TITLE
Update migrating-to-github-container-registry-for-docker-images.md

### LINKED ISSUE #5020: Inconsistent documentation about GCR support for GITHUB_TOKEN
### What is the current behavior?

https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images

currently states, under [Authenticating with the Container Registry](https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images#authenticating-with-the-container-registry):
> To authenticate to GitHub Container Registry within a GitHub Actions workflow, use the GITHUB_TOKEN for the best security and experience.

and

> GitHub Container Registry now supports GITHUB_TOKEN for easy and secure authentication in your workflows. If your workflow is using a personal access token (PAT) to authenticate to ghcr.io, then we highly recommend you update your workflow to use GITHUB_TOKEN.

Under [Updating your GitHub Actions Workflow](https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images#updating-your-github-actions-workflow), it conflictingly states:
> GitHub Container Registry does not support using GITHUB_TOKEN for your PAT so you must use a different custom variable, such as CR_PAT. For more information, see "Creating and storing encrypted secrets."

### What changes are you suggesting?

Given that GitHub Container Registry reportedly now supports `GITHUB_TOKEN`, presumably the updated workflow should recommend `GITHUB_TOKEN` instead of encouraging `CR_PAT` still.

### PR DESCRIPTION
**Pull request by danBamikiya**
This commit solves #5020. By preferring the use of the GITHUB_TOKEN instead of CR_PAT.

### Why:

**Closes #5020 **

### What's being changed:
github_docs_contribution_1

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ x ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ x ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@danBamikiya Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by lastthyme — APPROVED** [review]

**Review by lucalves — DISMISSED** [review]

File: content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md [hunk 1]
```diff
@@ -95,9 +95,9 @@ If you have a {% data variables.product.prodname_actions %} workflow that uses a
 
 2. In your {% data variables.product.prodname_actions %} workflow file, update the package url from `https://docker.pkg.github.com` to `ghcr.io`.
 
-3. Add your new {% data variables.product.prodname_container_registry %} authentication personal access token (PAT) as a GitHub Actions secret. The {% data variables.product.prodname_container_registry %} does not support using `GITHUB_TOKEN` for your PAT so you must use a different custom variable, such as `CR_PAT`. For more information, see "[Creating and storing encrypted secrets](/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)."
+3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT) as a GitHub Actions secret. {% data variables.product.prodname_container_registry %} now supports using `GITHUB_TOKEN` for your PAT. For more information, see "[Authenticating with the Github Token](/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."
```
Comment (hubwriter) [inline content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md:98]: I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

Comment (hubwriter) [inline content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md:98]: I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

**Review by hubwriter — COMMENTED** [review]
@danBamikiya - Many thanks for raising this pull request. 
Your changes are great, thanks. I've suggested one small change. 
If you'd like to commit my suggestion we'll get this merged. 
Or feel free to make an alternative change if you prefer.

Thanks once again for helping to improve the GitHub documentation. 👍

File: content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md (hunk 1 above)
Comment (danBamikiya) [reply content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md]: @hubwriter  Hi, I just did that!

**Review by hubwriter — APPROVED** [review]

**Comment by hubwriter** [comment]
@danBamikiya - Many thanks.
Merging this now. It will show up on the live site soon.
👏

**Pull request merged**